packages/flutter_tools/lib/src/linux/linux_doctor.dart

```diff
@@ -43,13 +43,21 @@ class LinuxDoctorValidator extends DoctorValidator {
   static const String kClangBinary = 'clang++';
   static const String kCmakeBinary = 'cmake';
   static const String kNinjaBinary = 'ninja';
+  static const String kPkgConfigBinary = 'pkg-config';
 
   final Map<String, Version> _requiredBinaryVersions = <String, Version>{
     kClangBinary: Version(3, 4, 0),
     kCmakeBinary: Version(3, 10, 0),
     kNinjaBinary: Version(1, 8, 0),
+    kPkgConfigBinary: Version(0, 29, 0),
   };
 
+  final List<String> _requiredLibraries = <String>[
+    'gtk+-3.0',
```

Comment:
Do we want to check the version of GTK here? If so this should be changed to `'gtk+-3.0 >= 3.22'` or similar. `pkg-config --exists gtk+-3.0 >= 3.22` or `pkg-config --exists gtk+3.0 --atleast-version=3.22` will return 1 if the version is not recent enough. You could use `--print-errors` to get error text or `--modversion if you want to do this yourself, i.e.

```
$ pkg-config --exists 'gtk+-3.0 >= 10' --print-errors
Requested 'gtk+-3.0 >= 10' but version of GTK+ is 3.24.18
```
```
$ pkg-config --modversion gtk+-3.0
3.24.18
```

I noticed we're not checking the version in the engine BUILD.gn - it should probably be done there too.